README.md

```diff
@@ -1 +1,65 @@
-#  Chart repo
+# OpenShift Helm Charts
+
+OpenShift Helm Charts is a repository hosting [Helm Charts](https://github.com/helm/helm) available by default with [OpenShift](https://www.openshift.com/). It contains popular technologies, tools and services. Helm Charts on this repository can be provided by the community, by partners or Red Hat. 
+
+Charts optionaly go through an automated RedHat OpenShift certification workflow, which garanties security compliance as well as best integration and experience with the platform.
+
+## Structure of the repository
+
+```
+.
```

Comment:
@baijum please confirm we can remove the charts folder. If too much work will change this to include charts folder.

Reply:
The `charts` folder is mentioned in all scripts. Probably I can search and remove it.

Reply:
Lets table this until we talk about the org rename as well. Adding charts directory to the structure